DevopsPlebGH/terraform-aws-sophos-firewall
# feature/subnet

- initial commit
- initial commit
- workflow file
- workflow test
- workflow test
- terraform backend init
- remote backend
- workflow run
- tfbackend path
- tfbackend
- matrix path
- remote backend
- workflow file
- testing local actions
- custom workflow
- worfklow
- workflow test
- re-running with @main ref
- removed workflows from feature branch
- workflow
- list and diff
- continue-on-error: true
- Steps outputs missing variable notation
- && for list output
- auto-pr
- autopr
- Title: Removed workflow dependency
- Added push to feature branches
- branch ignore
- branches-ignore
- checkout current ref and PR to dev
- autopr test
- gh pat token
- different autopr action
- autopr test
- auto-pr
- no pr template
- initial commit
- No PR created
- Error on workflow file
- ${{ github.head_ref }}
- auto-pr
- added assignee
- setting checkout ref to dev
- setting the base back to dev in pr action
- Using repo-sync action
- autopr from feature branch
- workflow if success() || failure()
- if success and/or failure
- removed format step
- format if lint fails
- check error and re-format
- fmt check
- outputs stderr
- fmtcheck stderr
- fmt check only
- lint should return failure
- lint step
- lint step
- lint
- continue on error
- check failure
- continue on failure
- continue on failure
- pre-commit hooks
- closing feature
- Title: AWS Public Subnet
- commit by jocelyn
- private subnet
